packages/botonic-react/src/webchat/components/attachment.jsx

```diff
@@ -1,21 +1,34 @@
-import React from 'react'
+import React, { useContext } from 'react'
 
 import AttachmentIcon from '../../assets/attachment-icon.svg'
-import { ROLES } from '../../constants'
+import { ROLES, WEBCHAT } from '../../constants'
+import { WebchatContext } from '../../contexts'
 import { Icon, IconContainer } from './common'
 
-export const Attachment = ({ onChange, accept }) => (
-  <IconContainer role={ROLES.ATTACHMENT_ICON}>
-    <label htmlFor='attachment' style={{ marginTop: 4 }}>
-      <Icon src={AttachmentIcon} />
-    </label>
-    <input
-      type='file'
-      name='file'
-      id='attachment'
-      style={{ display: 'none' }}
-      onChange={onChange}
-      accept={accept}
-    ></input>
-  </IconContainer>
-)
+export const Attachment = ({ onChange, accept, customAttacments }) => {
+  const { getThemeProperty } = useContext(WebchatContext)
+
+  const CustomAttachments = getThemeProperty(
+    WEBCHAT.CUSTOM_PROPERTIES.customAttachments,
+    undefined
+  )
```

Comment:
Since we're getting this theme property in the `webchat.jsx` file and we're already checking there if a custom attachment has been declared, why not leave the `Attachment` component as it is now and add the logic of deciding which one to render in the `webchat.jsx` file (as it is right now with the `SendButton` component)?

This way we keep the components much simpler. What do you think @Iru89 @vanbasten17?

Reply:
I am of the opposite opinion. I would create files for each component and manage the logic in that file. This way when viewing the webchat file I would only see where the EmojiPicker, Attachments or Sendbutton is. When you go into the file for each component you would see if there is a custom and what that component does.

Reply:
I have made a commit with WIP so that you can understand what I explained in the previous comment.

Reply:
I like this approach also :+1:! The most important thing is to have consistency by applying the same logic in all 3 components.